src/database/migrations/20230407081339-create-collections.js

```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from 'sequelize';
+
+/** @type {import('sequelize-cli').Migration} */
+export default {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('collections', {
+      id: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        primaryKey: true,
+        allowNull: false,
+        unique: true,
+      },
+      userId: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        unique: false,
```

Comment:
Why did you add another collection migration?

Reply:
is it a problem? I just even added somelines to modify the userId